### PR TITLE
dropdowns.less: Add space before IE8 workaround.

### PR DESCRIPTION
**Pull request by XhmikosR**
This in turn works around an issue with clean-css.

https://github.com/jakubpawlowicz/clean-css/issues/556

/CC @cvrebert @mdo

### CODE REVIEW

**Comment by cvrebert** [comment]
X-Ref: #16258

**Comment by cvrebert** [comment]
I've confirmed via Sauce that adding the space doesn't degrade the effectiveness of the IE8 fix.

**Comment by XhmikosR** [comment]
OK, so it's good to go then?

**Comment by XhmikosR** [comment]
BTW, don't forget to update the node_modules in bot and shrinkwrap for the new clean-css version.

**Comment by cvrebert** [comment]
The bot installs using the shrinkwrap file on every build, so we just need to reshrinkwrap.

**Comment by XhmikosR** [comment]
All right, done.

**Comment by XhmikosR** [comment]
Now, this PR is basically a consistency change since in all other cases we use `\9` there's a space before it.

File: less/dropdowns.less [hunk 1]
```diff
@@ -11,7 +11,7 @@
   margin-left: 2px;
   vertical-align: middle;
   border-top:   @caret-width-base dashed;
-  border-top:   @caret-width-base ~"solid\9"; // IE8
+  border-top:   @caret-width-base ~"solid \9"; // IE8
```
Comment (cvrebert) [inline less/dropdowns.less]: Perhaps we should use
```css
border-top:   @caret-width-base solid ~"\9"
```
?

Comment (XhmikosR) [inline less/dropdowns.less]: It results to the same CSS, not sure which one is preferable.

**Comment by cvrebert** [comment]
:shipit:

**Pull request merged**